File: secao-11-Linguagem-Java/exercicios/menor_de_tres.java
// Fazer um programa para ler três números inteiros. Em seguida, mostrar qual o menor dentre os três números lidos. Em caso de empate, mostrar apenas uma vez. 

import java.util.Locale;
import java.util.Scanner;

public class menor_de_tres {
    public static void main(String[] args) {
        
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int a, b, c, menor;

        System.err.print("Primeiro valor: ");
        a = sc.nextInt();
        System.err.print("Segundo valor: ");
        b = sc.nextInt();
        System.err.print("Terceiro valor: ");
        c = sc.nextInt();

        menor = a;
        if (b < menor) {
            menor = b;
        }
        if (c < menor) {
            menor = c;
        }

        System.out.println("Menor = " + menor);

        sc.close();        
    }
}

File: secao-12-Linguagem-C++/exercicios/menor-de_tres.cs
// Fazer um programa para ler três números inteiros. Em seguida, mostrar qual o menor dentre os três números lidos. Em caso de empate, mostrar apenas uma vez. 

using System;
using System.Globalization;

namespace menor_de_tres
{
    class Program
    {
        static void Main(String[] args)
        {
            CultureInfo CI = CultureInfo.InvariantCulture;

            int a, b, c, menor;

            Console.Write("Primeiro valor: ");
            a = int.Parse(Console.ReadLine());
            Console.Write("Segundo valor: ");
            b = int.Parse(Console.ReadLine());
            Console.Write("Terceiro valor: ");
            c = int.Parse(Console.ReadLine());

            menor = a;

            if(b < menor)
            {
                menor = b;
            }
            if (c < menor)
            {
                menor = c;
            }

            Console.WriteLine("MENOR = " + menor);
        }
    }
}
